packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -340,8 +340,15 @@ function keepParentInformedAboutDimensionChanges() {
       throw new Error("Main element should be an HTMLElement");
     }
 
-    const contentHeight = mainElement.offsetHeight;
-    const contentWidth = mainElement.offsetWidth;
+    const mainElementStyles = getComputedStyle(mainElement);
+    const contentHeight =
```

Comment:
offset doesn't include margin. So consider that as success page main element has margins

Reply:
Oh the joys :')